config/core/test-configs.yaml

```diff
@@ -608,6 +608,12 @@ device_types:
     class: arm-dtb
     boot_method: uboot
 
+  at91sam9g20ek:
+    mach: at91
+    class: arm-dtb
+    boot_method: uboot
+    dtb: "at91sam9g20ek.dtb"
```

Comment:
If the dtb name matches the device type name `at91sam9g20ek` then it shouldn't be necessary to specify it here.